app/services/discourse_rewind/action/new_user_interactions.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+# Tracks how much this user interacted with new users (created this year)
+# Shows veteran mentorship and community building behavior
+module DiscourseRewind
+  module Action
+    class NewUserInteractions < BaseReport
+      def call
+        year_start = Date.new(date.first.year, 1, 1)
+
+        # Find users who created accounts this year
+        new_user_ids =
+          User
```

Comment:
We should use the `real` scope for this:

https://github.com/discourse/discourse/blob/main/app/models/user.rb#L282-L292